certified-connectors/Easyvista_Service._Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,17505 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "Different endpoints for creating/updating/deleting in the Easyvista Service Manager database / EasyVista Service Manager Modification May 2023",
```

Comment:
@ezv-compteintegration is it possible to remove "/ EasyVista Service Manager Modification May 2023" from the description?